tests/unit/test_service_steps.py

```diff
@@ -768,10 +769,51 @@ def test_eks_create_cluster_step_creation_sync():
 
 
 @patch.object(boto3.session.Session, 'region_name', 'us-east-1')
-def test_eks_delete_cluster_step_creation():
-    step = EksDeleteClusterStep("Delete Eks cluster", wait_for_completion=False, parameters={
-        'Name': 'MyCluster'
-    })
+def test_eks_create_cluster_step_creation_wait_for_completion():
+    step = EksCreateClusterStep("Create Eks cluster - WaitForCompletion",
+                                integration_pattern=IntegrationPattern.WaitForCompletion,
+                                parameters={
+                                    'Name': 'MyCluster',
+                                    'ResourcesVpcConfig': {
+                                        'SubnetIds': [
+                                            'subnet-00000000000000000',
+                                            'subnet-00000000000000001'
+                                        ]
+                                    },
+                                    'RoleArn': 'arn:aws:iam::123456789012:role/MyEKSClusterRole'
+                                })
+
+    assert step.to_dict() == {
+        'Type': 'Task',
+        'Resource': 'arn:aws:states:::eks:createCluster.sync',
+        'Parameters': {
+            'Name': 'MyCluster',
+            'ResourcesVpcConfig': {
+                'SubnetIds': [
+                    'subnet-00000000000000000',
+                    'subnet-00000000000000001'
+                ]
+            },
+            'RoleArn': 'arn:aws:iam::123456789012:role/MyEKSClusterRole'
+        },
+        'End': True
+    }
+
+
+@patch.object(boto3.session.Session, 'region_name', 'us-east-1')
+def test_eks_create_cluster_step_creation_wait_for_task_token_raises_error():
+    with pytest.raises(ValueError):
```

Comment:
nit: can we assert the message as well? I think `pytest.raises` accepts a `match=<message>` - [documentation](https://docs.pytest.org/en/stable/reference.html?highlight=raises#pytest.raises)

multiple occurrences in this PR where we can do this...

Reply:
Yes - adding a validation on the error message will ensure no breaking changes are introduced

Added assert on error message in the last commit!